### ISSUE TITLE
PHP 8.2 causes errors and issues with remote poller

### ISSUE THREAD

**Issue by jlrcontegix** [Cacti/cacti#5242]
# Describe the bug

PHP 8.2 setup on a remote poller produces the following errors in cacti.log. This occurs with some regularity, but is reproducible by going to a device that is being polled with a remote poller that is using PHP 8.2.x.  In the Web UI an error is displayed about contacting the device.  When downgrading the remote poller to PHP 7.4.x the errors go away and functionality is as expected.

```
ERROR PHP DEPRECATED: Creation of dynamic property phpsnmp\SNMP::$bulk_walk_size is deprecated in file: /var/www/html/lib/snmp.php  on line: 87
CMDPHP PHP ERROR Backtrace:  (/remote_agent.php[78]:ping_device(), /remote_agent.php[293]:api_device_ping_device(), /lib/api_device.php[1492]:cacti_snmp_session(), /lib/snmp.php[87]:CactiErrorHandler())
ERROR PHP DEPRECATED: Creation of dynamic property phpsnmp\SNMP::$value_output_format is deprecated in file: /var/www/html/lib/snmp.php  on line: 445
CMDPHP PHP ERROR Backtrace:  (/remote_agent.php[72]:run_remote_data_query(), /remote_agent.php[439]:run_data_query(), /lib/data_query.php[106]:query_snmp_host(), /lib/data_query.php[822]:cacti_snmp_session_walk(), /lib/snmp.php[445]:CactiErrorHandler())
```

## To Reproduce

Install fresh 1.2.23 instance with PHP 8.2.x and configure as a remote poller (probably same issue with main poller instances, but we have not upgraded them to PHP 8).  Edit/open a device being polled by this remote poller and see the errors in cacti.log and UI.

**Comment by TheWitness** [comment]
I'm pretty sure this is fixed in the 1.2.x branch.  Please test there.

**Comment by TheWitness** [comment]
Resolved here.

https://github.com/Cacti/cacti/issues/5140
